src/components/nc-spinner.vue

```diff
@@ -0,0 +1,162 @@
+<template>
```

Comment:
Siguiendo la sintaxis de vuetify, cambiaría el nombre del componente a algo como nc-progress, dado que el spinner sería el icono circular que muestra el componente, pero también podemos necesitar mostrar progreso lineal asociado a un porcentaje de carga.

Reply:
Igual es mejor dejarlo como spinner o loading? Tiene pinta de que en algún momento podra caher una barra de progreso y ya tendremos el nombre cogido

Reply:
Estoy de acuerdo en que hay que cambiar el nombre, pero como Alberto también pienso que puede ser que en el futuro haya un componente "barra de progreso". En la librería de "semantic-ui-vue" [https://semantic-ui-vue.github.io/#/elements/loader](url) Tienen un **loader**.. ¿Qué opináis de un nc-loader?

Reply:
> Estoy de acuerdo en que hay que cambiar el nombre, pero como Alberto también pienso que puede ser que en el futuro haya un componente "barra de progreso". En la librería de "semantic-ui-vue" [https://semantic-ui-vue.github.io/#/elements/loader](url) Tienen un **loader**.. ¿Qué opináis de un nc-loader?

Lo veo bien

Reply:
yo tmb...nc-loader adjudicado

Reply:
Modificado en el PR